kse/src/main/java/org/kse/crypto/x509/X509Ext.java

```diff
@@ -2173,36 +2146,34 @@ private static String getSemanticInformationValueString(QcStatementType qcStatem
 
         StringBuilder sb = new StringBuilder();
 
```

Comment:
Why was this indent removed?

Reply:
Because `baseIndentLevel` was in this case always 1.
And never use on recursion.
And then, after factorisation, the Indent is now on line 2067.